packages/apps-engine/deno-runtime/lib/accessors/builders/RoomBuilder.ts

```diff
@@ -160,4 +184,14 @@ export class RoomBuilder implements IRoomBuilder {
     public getRoom(): IRoom {
```

Comment:
Should we use this opportunity to freeze this one? Or do you consider that to be a breaking change?